packages/now-next/src/index.ts

```diff
@@ -990,6 +1003,16 @@ export const build = async ({
       ...staticFiles,
       ...staticDirectoryFiles,
     },
+    /*
+      Desired routes order
+      - Runtime headers
+      - User headers and redirects
+      - Runtime redirects
+      - Runtime routes
+      - Check filesystem, if nothing found continue
+      - User rewrites
+      - Builder rewrites
+    */
     routes: [
```

Comment:
Do you need to add `headers` here too?

Reply:
Yeah, looks like they got removed when resolving merge conflict 😅